pkg/plugin/v3/api.go

```diff
@@ -137,6 +137,10 @@ func (p *createAPIPlugin) Validate() error {
 		return err
 	}
 
+	if p.resource.Group == "" && p.config.Domain == "" {
```

Comment:
I think this check is redundant, given that `p.resource.Validate()` never allows p.Config.Domain to be an empty string in the first place?

Reply:
@gabbifish IMO this is a necessary check, while initializing a project (kb init) you can provide a domain which is empty. 

Validate() function doesn't check the value of domain, it only checks the group, version, kind and related values which are provided in create api command. (kb create api). Domain is provided at the init level. Hence this check is necessary to know a user do not provide domain and group both as empty.